core/controllers/blog_admin_test.py

```diff
@@ -141,193 +136,157 @@ def setUp(self) -> None:
+            'new_platform_parameter_values': {
+                (
+                    platform_parameter_list.ParamNames.
+                    MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST.value
+                ): new_platform_parameter_value,
             }
         }
         response_dict = self.post_json(
             '/blogadminhandler', payload, csrf_token=csrf_token,
             expected_status_int=400)
         self.assertEqual(
-            response_dict['error'], 'Schema validation for \'new_config_'
-            'property_values\' failed: Validation failed: is_at_least'
-            ' ({\'min_value\': 1}) for object 0'
-        )
+            response_dict['error'], 'Schema validation for \'new_platform_'
+            'parameter_values\' failed: The value of max_number_of_tags_'
+            'assigned_to_blog_post should be greater than 0, it is -2.')
 
-    def test_raise_error_for_updating_to_negative_value_for_max_tags(
-        self
-    ) -> None:
+    def test_invalid_value_for_platform_param_raises_error(self) -> None:
         self.login(self.BLOG_ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
-        new_config_value = -2
+        new_platform_parameter_value = 'string'
 
         response_dict = self.get_json('/blogadminhandler')
-        response_config_properties = response_dict['config_properties']
+        response_platform_parameters = response_dict['platform_parameters']
         self.assertDictContainsSubset({
             'value': 10,
-        }, response_config_properties[
-            config_domain.MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST.name])
+        }, response_platform_parameters[
+            platform_parameter_list.ParamNames.
+            MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST.value]
+        )
 
         payload = {
-            'action': 'save_config_properties',
-            'new_config_property_values': {
-                config_domain.MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST.name: (
-                    new_config_value),
+            'action': 'save_platform_parameters',
+            'new_platform_parameter_values': {
+                (
+                    platform_parameter_list.ParamNames.
+                    MAX_NUMBER_OF_TAGS_ASSIGNED_TO_BLOG_POST.value
+                ): new_platform_parameter_value,
             }
         }
         response_dict = self.post_json(
             '/blogadminhandler', payload, csrf_token=csrf_token,
             expected_status_int=400)
         self.assertEqual(
-            response_dict['error'], 'Schema validation for \'new_config_'
-            'property_values\' failed: Validation failed: is_at_least'
-            ' ({\'min_value\': 1}) for object -2'
-        )
+            response_dict['error'], 'Schema validation for \'new_platform_'
+            'parameter_values\' failed: The value of platform parameter '
+            'max_number_of_tags_assigned_to_blog_post is of type string, '
+            'expected it to be number')
```

Comment:
...expected it to be of type number

(Better: have single quotes around 'string' and 'number')

Reply:
Updated